functions/uploader/entities/entities.js

```diff
@@ -10,12 +10,20 @@ const MAX_ENTITY = 30000;
 const MAX_REQUEST = 1000;
 
 const basicHeaderRequest = {
-  'content-type': 'application/json; charset=UTF-8',
-  'authorization': 'BEARER a55db7f7ed0a4b52a0982b22e2428f18'
+  `content-type`: `application/json; charset=UTF-8`,
+  `authorization`: `BEARER ${process.env.AUTHORIZATION}`
```

Comment:
`AUTHORIZATION` is too general we should use something more specific, for example: `DIALOG_FLOW_DEV_TOKEN`